packages/p2p/src/components/composite-calendar/calendar-side-list.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { toMoment } from '@deriv/shared';
+import ListItem from './list-item.jsx';
+import './calendar-side-list.scss';
+
+const isActive = (from, to, flag) => {
+    if (flag === 0) {
+        return toMoment().endOf('day').unix() === to && from === null;
+    }
+    return Math.ceil(to / 86400) - Math.ceil(from / 86400) === flag;
+};
+
+const SideList = ({ items, from, to }) => (
```

Comment:
```suggestion
const CalendarSideList = ({ items, from, to }) => (
```

could we keep the naming conventions similar to the file name?